deployment/team.go

```diff
@@ -0,0 +1,106 @@
+package deployment
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/astronomer/astro-cli/houston"
+	"github.com/astronomer/astro-cli/messages"
+	"github.com/astronomer/astro-cli/pkg/printutil"
+	log "github.com/sirupsen/logrus"
+)
+
+var errHoustonInvalidDeploymentTeams = errors.New(messages.HoustonInvalidDeploymentTeams)
+
+// TeamsList returns a list of teams with deployment access
+func ListTeamRoles(deploymentID string, client houston.ClientInterface, out io.Writer) error {
+	deploymentTeams, err := client.ListDeploymentTeamsAndRoles(deploymentID)
+	if err != nil {
+		log.Error(err)
```

Comment:
Do we need to log this error here ?

The error will already be displayed to the customer (as the command exits)

Reply:
Yes I agree. I've removed it